docs/dev/system_keyspace.md

```diff
@@ -307,6 +307,35 @@ CREATE TABLE system.cluster_status (
 
 Implemented by `cluster_status_table` in `db/system_keyspace.cc`.
 
+## system.load_per_node
+
+Contains information about the current tablet load with node granularity.
+Can be queried on any node, but the data comes from the group0 leader.
+Reads wait for group0 leader to be elected and load balancer stats to become available.
+
+Schema:
+```cql
+CREATE TABLE system.load_per_node (
+    node uuid PRIMARY KEY,
+    dc text,
+    rack text,
+    storage_allocated_load bigint,
+    storage_allocated_utilization double,
+    storage_capacity bigint,
+    tablets_allocated bigint,
+    tablets_allocated_per_shard double
```

Comment:
Idea: make these static column, and add a clustering key for per-table information. This will allow us to detect table imbalance.

Reply:
I think it's better to create a separate table for per-table stats to make queries which don't need this level cheaper and to not obfuscate the output.